rpm/generic/zfs.spec.in

```diff
@@ -363,12 +381,25 @@ systemctl --system daemon-reload >/dev/null || true
 %endif
 
 %files
+# Core utilities
 %{_sbindir}/*
-%{_bindir}/*
-%{_libexecdir}/%{name}
+%{_bindir}/raidz_test
+%{_bindir}/zgenhostid
+# Optional Python 2/3 scripts
+%{_bindir}/arc_summary.py
+%{_bindir}/arcstat.py
+%{_bindir}/dbufstat.py
+%if 0%{?_pypkg_version} == 3 || 0%{?_pypkg_version} >= 34
+%{_bindir}/arc_summary3.py
```

Comment:
Can we please have Python programs that are installed into `%_bindir` not suffixed with `.py`? That way Python won't attempt to generate bytecode for it and write byte compilation objects (`*.py[co]` files).

Reply:
Yes, thanks for the reminder.  Now is definitely the time to do it.